sr/comp/cli/show_match_scores.py

```diff
@@ -0,0 +1,167 @@
+__description__ = "Show the game and league points achieved for each match"
+
+from typing import Dict, List, Union, NamedTuple
+from sr.comp.types import TLA, GamePoints, ArenaName, MatchNumber
+from sr.comp.scores import LeaguePosition
+
+
+class MatchCorner(NamedTuple):
+    tla: TLA
+    ranking: Union[LeaguePosition, str]  # to allow "???" to be used for unknown scores
+    game: Union[GamePoints, str]
+    league: Union[int, str]
+
+
+class MatchResult(NamedTuple):
+    num: MatchNumber
+    arena: ArenaName
+    display_name: str
+    corners: Dict[int, MatchCorner]
+
+
+def collect_match_info(comp, match) -> MatchResult:
```

Comment:
I expect `mypy` wants the arguments to be typed too.

Reply:
I've added types in e0a900a0aa3a4d5db9791bafe02334bf57082d37 but I have a type error with the degroup command.
```
sr/comp/cli/show_match_scores.py:47: error: Argument 1 to "degroup" has incompatible type "Mapping[RankedPosition, Set[TLA]]"; expected "Mapping[Optional[LeaguePosition], Iterable[TLA]]"  [arg-type]
```